packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/ClientEventType.java

```diff
@@ -29,6 +29,7 @@ public enum ClientEventType {
 	TASK_GOLLM_DATASET_STATISTICS,
 	TASK_MIRA_AMR_TO_MMT,
 	TASK_MIRA_GENERATE_MODEL_LATEX,
+	TASK_MIRA_SIMPLIFY_MODEL,
```

Comment:
This event type `TASK_MIRA_SIMPLIFY_MODEL` is not being used.

Reply:
yes, from my understanding it will be used in the next PR. 
I will remove it from here as it can be added back easily enough